flake.nix

```diff
@@ -5,10 +5,16 @@
 
   outputs = { self, nixpkgs, flake-utils }:
     flake-utils.lib.eachDefaultSystem (system: let
-      pkgs = nixpkgs.legacyPackages.${system};
+      pkgs = import nixpkgs {
+        inherit system;
+        overlays = [ self.overlay ];
+      };
```

Comment:
This should be not necessary. It makes evaluation more expensive by re-importing nixpkgs.

Reply:
Gosh whoops! Let me fix this.